tests/stress_tests/CMakeLists.txt

```diff
@@ -9,12 +9,19 @@ if (CMAKE_BUILD_TYPE STREQUAL "")
     set(CMAKE_BUILD_TYPE "Release")
 endif()
 
-# Define directory where artifacts will be placed
-set(OUTPUT_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
+set(OpenVINO_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../")
 
-find_package(InferenceEngineDeveloperPackage REQUIRED)
+# Search OpenVINO Inference Engine installed
+find_package(InferenceEngine)
 
-set(OpenVINO_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../")
+if(NOT InferenceEngine_FOUND)
+    # Define directory where artifacts will be placed
+    set(OUTPUT_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
where the artifacts will be placed w/o dev package?

Reply:
All artifacts will be placed in <tests_build_dir> with folders layout like .cpp are placed (e.g. for timetest_infer it's openvino/tests/time_tests/build/src/timetests/timetest_infer). Later with `make install` artifacts will be placed in tests/  folder